shone/chemistry/fastchem.py

```diff
@@ -201,7 +217,180 @@ def mmr_mmw(self):
             for each species.
         """
         vmr = self.vmr()
-        weights = self.get_weights()
-        mmr_mmw = vmr * weights[None, :]
+        weights_amu = self.get_weights()
+        mmr_mmw = vmr * weights_amu[None, :]
 
         return mmr_mmw
+
+
+def round_in_log(x):
+    log_floor = np.floor(np.log10(x))
+    pre_exponent = np.round(x * 10 ** -log_floor, 1)
+    return pre_exponent * 10 ** log_floor
+
```

Comment:
Should this be jittable and does it need to be jnp?

Reply:
It doesn't need to be, it's just a convenience function for getting round and roughly log-spaced numbers from a true log-spaced array.